src/libs/Navigation/types.ts

```diff
@@ -331,10 +331,6 @@ type ProcessMoneyRequestHoldNavigatorParamList = {
 };
 
 type PrivateNotesNavigatorParamList = {
-    [SCREENS.PRIVATE_NOTES.VIEW]: {
```

Comment:
Why was this page removed?

Reply:
It is not required anymore so removed. https://expensify.slack.com/archives/C01GTK53T8Q/p1705500633296999